tasks/RedHat.yml

```diff
@@ -19,6 +19,12 @@
     - ansible_distribution == "Amazon"
     - ansible_distribution_major_version == "NA"
 
+- name: "Fedora | Set ansible_distribution_major_version to 7 when Fedora"
+  set_fact:
+    ansible_distribution_major_version: 7
```

Comment:
Idea to add in the `default.yml` a property named `zabbix_agent_conf_dir` and give it value `/etc/zabbix` and with this `set_fact`, we set it for Fedora to `/etc`? Then we only have to change the template dest path for the `zabbix-agent.conf` and you won't have to use the `copy` module for Fedora.

Or am I missing something that would not help in this case?